src/templates/data/repository/sqlserver/repository.ejs

```diff
@@ -0,0 +1,21 @@
+const { Repository } = require("@herbsjs/herbs2knex")
+const { <%= props.name.pascalCase %> } = require('../../../domain/entities')
+
+module.exports = class <%= props.name.pascalCase %>Repository extends Repository {
```

Comment:
this is not the same template that exists into postgres folder? i think we can reuse it, so just bring the template for "repository folder" and use it for postgres and sqlserver